specification/api-metrics-user.md

```diff
@@ -9,9 +9,23 @@ TODO: Add a table of contents.
 
 ## Overview
 
-Metric instruments are the entry point for application and framework
-developers to instrument their code using counters, gauges, and
-measures.
+Metric instruments are the entry point for application and framework developers to instrument their code using counters, gauges, and measures.
+Metrics are created by calling methods on a `Meter` which is in turn created by a global `MeterFactory`.
+
+### Obtaining a Meter
+
+New `Meter` instances can be created via a `MeterFactory` and its `getMeter` method.
+`MeterFactory`s are generally expected to be used as singletons.
+Implementations SHOULD provide a single global default `MeterFactory`. The `getMeter` method expects two string arguments:
+
+- `name` (required): This name must identify the instrumentation library (also referred to as integration, e.g. `io.opentelemetry.contrib.mongodb`)
+and *not* the instrumented library.
```

Comment:
Since you added line breaks, please also indent the lines belonging to the bullet point with two spaces (try running markdownlint).

Reply:
markdownlint is fine with it but I'll indent it if you prefer

Reply:
I prefer to have each bullet/paragraph/block on a single line so that each person's editor can wrap in whatever way they prefer and not where the author happened to insert a line break.